modules/update-service-overview.adoc

```diff
@@ -48,5 +48,18 @@ The OpenShift Update Service is composed of an Operator and one or more applicat
 [NOTE]
 ====
 During the upgrade process, nodes in the cluster might become temporarily unavailable. The `MachineHealthCheck` might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, remove any `MachineHealthCheck` resource that you have deployed before updating the cluster.
```

Comment:
I think we should change from removing to pausing the MHC here

Reply:
instead of the extra paragraph about pausing below

Reply:
@slintes Makes sense. We [added that note in 4.8 docs about stopping the check](https://github.com/openshift/openshift-docs/pull/35373). I wondered if we should retain it and add highlight/clarify that the mechanism changes in 4.9.
Also adding @JoelSpeed and @mburke5678 since they reviewed the previous PR.